tools/format.py

```diff
@@ -48,14 +48,17 @@ def main():
         prettier()
 
 
-def qrun(cmd, env=None):
-    run(cmd, quiet=True, env=env)
+def qrun(cmd, cwd=None, env=None):
+    run(cmd, quiet=True, cwd=cwd, env=env)
 
 
 def clang_format():
     print "clang_format"
-    qrun([clang_format_path, "-i", "-style", "Google"] +
-         find_exts(["core"], [".cc", ".h"]))
+    qrun(
+        [sys.executable, clang_format_path, "-i", "-style", "Google"] +
+        find_exts(["core"], [".cc", ".h"],
+                  skip=["core/libdeno/build", "core/libdeno/third_party"]),
+        env={"CHROMIUM_BUILDTOOLS_PATH": "third_party/v8/buildtools"})
```

Comment:
What’s the CHROMIUM_BUILDTOOLS_PATH for ?

Reply:
It's for fixing:

```
Problem while looking for clang-format in Chromium source tree:
Could not find checkout in any parent of the current path.
Set CHROMIUM_BUILDTOOLS_PATH to use outside of a chromium checkout.
```

But honestly don't ask me why it worked with the old folder layout without setting it.

Reply:
This should fix it: https://github.com/denoland/deno/pull/2938/commits/e74a0cae91d3df6e2434315d690fd08c3cb6598e
(I added this patch to your branch)

Reply:
@piscisaureus: Thanks, this definitely explains the inconsistency with symlinks I've observed on Windows. However it now requires some additional fixing, because we now have to really ignore all symlinks with `find_exts`. But unfortunately is doesn't fix the need for the `CHROMIUM_BUILDTOOLS_PATH` environment variable. (This also wasn't a Windows only issue as I can reproduce this on macOS as well).